emulators/melonds/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+PortGroup           qt5 1.0
+
+github.setup        Arisotura melonDS 0.9.2
+revision            0
+
+name                melonds
+platforms           darwin
+license             GPL-3
+maintainers         {gmail.com:audvare @Tatsh} openmaintainer
+categories          emulators games
+description         Nintendo DS emulator.
+long_description    {*}${description}
+
+checksums           rmd160  87d6585506c8abdae6bf7417708e1b30578b1927 \
+                    sha256  d20ce23725e63be5c60d42c87ed504015aa57ddf57c5161302c98e1837e9acae \
+                    size    1331032
+
+depends_lib-append  port:gettext \
+                    path:lib/pkgconfig/glib-2.0.pc:glib2 \
+                    port:libarchive \
+                    port:libepoxy \
+                    port:libiconv \
+                    port:libsdl2 \
+                    port:libslirp
+
+destroot {
+    move ${workpath}/build/melonDS.app ${destroot}${applications_dir}
+}
+
+if {${os.major} < 18} {
```

Comment:
```suggestion
if {${os.platform} eq “darwin” && ${os.major} < 18} {
```

Reply:
Smart quotes in this suggestion tricked me.